integration_test/third_party_apps_data/agent/ops-agent/linux/enable_zookeeper

```diff
@@ -0,0 +1,34 @@
+# Configures Ops Agent to collect telemetry from the app and restart Ops Agent.
+
+set -e
+
+sudo tee /etc/google-cloud-ops-agent/config.yaml > /dev/null << EOF
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+metrics:
+  receivers:
+    zookeeper:
+      type: zookeeper
+      endpoint: localhost:2181
```

Comment:
I thought `endpoint` and `collection_interval` were removed from `enable_zookeeper`, which is the expectation so that we test the bare minimum configuration in the sample config. Did they somehow get added back during a merge conflict resolution?

Reply:
Thanks for addressing this!